<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Slideshow</class>
 <widget class="QWidget" name="Slideshow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slideshow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="ImagePath" name="imgPath">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>721</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="DotsContainer" name="dotsContainer" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>851</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton::indicator{
width: 7px;
height: 7px;
border: none;
}

QRadioButton::indicator::unchecked {
image: url(:/images/indicator_unchecked.png);
}

QRadioButton::indicator::checked{
image: url(:/images/indicator_checked.png);
}</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>851</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>408</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>438</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="ImageView" name="imageView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>854</width>
     <height>480</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="backgroundBrush">
    <brush brushstyle="SolidPattern">
     <color alpha="255">
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
    </brush>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="moveBtnForward">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>320</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moveBtnBackward">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="flavText3">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>580</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Marco Di Rienzo</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dirBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="flavText1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Images directory:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="browseBtn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QLabel" name="flavText2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Images count:</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="imgCount">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>20</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="flavText4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Image path:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DotsContainer</class>
   <extends>QWidget</extends>
   <header>dotscontainer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header>ImageView.h</header>
  </customwidget>
  <customwidget>
   <class>ImagePath</class>
   <extends>QLabel</extends>
   <header>ImagePath.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
